src/helpers/QueryHelper.ts

```diff
@@ -9,4 +40,373 @@ export default class QueryHelper {
+            Child: QueryHelper.equalsFilterExpression(field, value),
+        };
+
+        return notEqualsExpression;
+    };
+
+    static andFilterExpression = (children: TApiQueryFilterExpression[]) => {
+        const andFilterExpression: IApiQueryAndFilterExpressionOperator = {
+            __type: 'AndFilterExpressionOperator:#EQ',
+            Children: children,
+        };
+
+        return andFilterExpression;
+    };
+
+    static orFilterExpression = (children: TApiQueryFilterExpression[]) => {
+        const orFilterExpression: IApiQueryOrFilterExpressionOperator = {
+            __type: 'OrFilterExpressionOperator:#EQ',
+            Children: children,
+        };
+        return orFilterExpression;
+    };
+
+    static lessFilterExpression = (field: TApiQueryField, value: TFilterValue) => {
+        const lessFilterExpression: IApiQueryLessFilterExpressionPredicate = {
+            __type: 'LessFilterExpressionPredicate:#EQ',
+            Field: field,
+            Value: value,
+        };
+
+        return lessFilterExpression;
+    };
+
+    static lessOrEqualFilterExpression = (field: TApiQueryField, value: TFilterValue) => {
+        const lessOrEqualFilterExpression: IApiQueryLessOrEqualFilterExpressionPredicate = {
+            __type: 'LessOrEqualFilterExpressionPredicate:#EQ',
+            Field: field,
+            Value: value,
+        };
+
+        return lessOrEqualFilterExpression;
+    };
+
+    static greaterFilterExpression = (field: TApiQueryField, value: TFilterValue) => {
+        const greaterFilterExpression: IApiQueryGreaterFilterExpressionPredicate = {
+            __type: 'GreaterFilterExpressionPredicate:#EQ',
+            Field: field,
+            Value: value,
+        };
+
+        return greaterFilterExpression;
+    };
+
+    static greaterOrEqualFilterExpression = (field: TApiQueryField, value: TFilterValue) => {
+        const greaterFilterExpression: IApiQueryGreaterOrEqualFilterExpressionPredicate = {
+            __type: 'GreaterOrEqualFilterExpressionPredicate:#EQ',
+            Field: field,
+            Value: value,
+        };
+
+        return greaterFilterExpression;
+    };
+
+    static likeFilterExpression = (field: TApiQueryField, value: TFilterValue) => {
+        const likeFilterExpression: IApiQueryLikeFilterExpressionPredicate = {
+            __type: 'LikeFilterExpressionPredicate:#EQ',
+            Field: field,
+            Value: `%${value}%`,
+        };
+
+        return likeFilterExpression;
+    };
+
+    static inFilterExpression = (field: TApiQueryField, values: TFilterValue[]) => {
+        const inFilterExpression: IApiQueryInFilterExpressionPredicate = {
+            __type: 'InFilterExpressionPredicate:#EQ',
+            Field: field,
+            Value: values
+        };
+
+        return inFilterExpression;
+    };
+
+    static getSuggestedContactsQuery = (emailContact: TEmailAddressDetails, queryFields: TApiQueryField[]) => {

```

Comment:
Odebereš pls tyhle suggested a getContactSearchParts? Ty jsou hodně app-specifické a navíc se ani nepoužívají - nenašel jsem použití. Ty ano?